frontend/package.json

```diff
@@ -36,6 +36,7 @@
     "lodash-es": "^4.17.21",
     "node-emoji": "^1.10.0",
     "preact": "^10.5.13",
+    "qrcode.react": "^1.0.1",
```

Comment:
We connect lib for QR code into bundle even if it's not enabled and it adds weight to it.
I would suggest finding the smallest lib for that or even better generate code on the backend.

like so:
`<img src="https://localhost/api/telegram/qr" />`

cc @paskal

Reply:
Fixed in e6b5355